addons/web/static/src/js/core/utils.js

```diff
@@ -354,7 +354,7 @@ var utils = {
      * @param {Number} size number of bytes
      */
     human_size: function (size) {
-        var units = _t("Bytes|Kb|Mb|Gb|Tb|Pb|Eb|Zb|Yb").split('|');
```

Comment:
I know it's tempting to be "more correct", but using the technical IEC units will just confuse users, as they've never heard of those. Users don't count bytes manually and don't care about "more correct" units. The only reference users have about file size is what [their OS tells them](https://en.wikipedia.org/wiki/Binary_prefix#Operating_systems). Plus this number is only useful as a rough estimation anyway. So I would stick with the commonly accepted units of "kB" or "KB" for 1024 bytes as an acceptable trade-off for most users, because that's what Windows displays.

But most importantly, I would not change that in stable at all, as it's not strictly necessary, and our end-users are used to it. Feel free to make a separate PR for changing "Kb" into "KB" in master ;-)

PS: https://ux.stackexchange.com/a/13850 and https://blog.codinghorror.com/gigabyte-decimal-vs-binary/

Reply:
We are letting hard drive manufacturer and the ISPs lie to users if everybody uses the same notation for different things in the same context; but ok, I get your point.